File: cop3223H/mathFunctions.c
//Federico Baron
//8/26/18
//Solves a quadratic formula with real roots

#include <stdio.h>
#include <math.h>

int main(void){
	double a,b,c;
	printf("Please enter a, b and c from your quadratic.\n");
	scanf("%lf%lf%lf", &a,&b,&c);

	//calculate the discriminant
	double discriminant = pow(b,2)-4*a*c;
	return 0;

	//use that to calculate the two roots
	double root1 = (-b + sqrt(discriminant))/(2*a);
	double root2 = (-b - sqrt(discriminant))/(2*a);

	//print the roots
	printf("Roots are %.12lf and %.12lf.\n", root1, root2);
	return 0;
}

File: cop3223H/Code/mathFunctions.py
#Federico Baron
#8/27/18
import math

#x = math.ceil(3.7)
#print(x)
#math.fabs is absolute value
#math.factorial
#math.pow
#math.sqrt

#Solves a quadratic with real roots.

# Read in the coefficients to the quadratic
a=float(input("Please enter a from your quadratic equation.\n"))
b=float(input("Please enter b from your quadratic equation.\n"))
c=float(input("Please enter c from your quadratic equation.\n"))

#Calculate the discriminant
discriminant = math.pow(b,2) -4*a*c

#Calculate both roots
root1 = (-b + math.sqrt(discriminant))/(2*a)
root2 = (-b - math.sqrt(discriminant))/(2*a)

#print out the roots
print("The roots of your quadratic are",root1,"and",root2)
